internal/rbd/rbd_util.go

```diff
@@ -355,13 +353,12 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 	klog.V(4).Infof(util.Log(ctx, "rbd: delete %s using mon %s, pool %s"), image, pOpts.Monitors, pOpts.Pool)
 
 	var ioctx *rados.IOContext
-	ioctx, err = pOpts.conn.GetIoctx(pOpts.Pool)
+	err = pOpts.openIoctx()
```

Comment:
hmm, in a follow-up change you call `openIoctx()`. Still easier to call `open()`